apps/explorer/priv/repo/migrations/20191011140922_create_celo_account.exs

```diff
@@ -48,10 +49,33 @@ defmodule Explorer.Repo.Migrations.CreateCeloAccount do
     create table(:celo_validator_history) do
       add(:index, :integer, null: false)
       add(:block_number, :integer, null: false)
-      add(:validator_address, :bytea, null: false)
+      add(:address, :bytea, null: false)
       timestamps(null: false, type: :utc_datetime_usec)
     end
 
-    create(index(:celo_validator_history, [:validator_address, :block_number, :index], unique: true))
+    create(index(:celo_validator_history, [:block_number, :index], unique: true))
+
+    if Mix.env() != :test do
```

Comment:
Is this a common pattern?

Reply:
Let's add an issue for removing these.

Reply:
https://github.com/celo-org/celo-monorepo/issues/1846